src/models/User.ts

```diff
@@ -106,36 +132,76 @@ const userSchema = new Schema({
       ref: "Event",
     },
   ],
-  userType: {
+  educationGrade: {
+    type: String,
+    enum: [
+      "NO_GRADE",
+      "PRE_KG",
+      "KG",
+      "GRADE_1",
+      "GRADE_2",
+      "GRADE_3",
+      "GRADE_4",
+      "GRADE_5",
+      "GRADE_6",
+      "GRADE_7",
+      "GRADE_8",
+      "GRADE_9",
+      "GRADE_10",
+      "GRADE_11",
+      "GRADE_12",
+      "GRADUATE",
+      null,
```

Comment:
I'm not sure if this is the right syntax for this. If the enum is nullable shouldn't it be `required: false`?

Reply:
`required: false` is the default for all fields, I believe. 
I tried doing this without appending null at enum, but it was not working.

Reply:
Is this resolved?

Reply:
Resolved from my side. 
I found an issue in Mongoose related to this.
https://github.com/Automattic/mongoose/issues/3044
 
 @xoldyckk Any more thoughts on this?

Reply:
If it works then no problem. I don't use mongodb/mongoose so I got no clue about all these gotchas.